server/plugin/subscriptions.go

```
@@ -302,6 +366,26 @@ func (p *Plugin) StoreSubscriptions(s *Subscriptions) error {
 	return nil
 }
 
+func (p *Plugin) GetSubscribedChannelsForOrg(org string) []*Subscription {
+	subs, err := p.GetSubscriptions()
+	if err != nil {
+		return nil
+	}
```

We probably want to surface this error, either by logging or returning the error to the caller